dune-project

```diff
@@ -1 +1 @@
-(lang dune 2.7)
```

Comment:
We're not using any new features, so we don't need to do this now. It would be better if we don't force everybody to upgrade immediately.

Reply:
Reverted.